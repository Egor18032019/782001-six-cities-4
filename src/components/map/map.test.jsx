import React from "react";
import renderer from "react-test-renderer";
import Map from "./map.jsx";

const PLACE = {
  id: 2,
  city: `Amsterdam`,
  type: `Private room`,
  description: `Wood and Stone`,
  prise: 120,
  isBookmark: false,
  isPremium: false,
  rating: 22,
  coordinate: [52.369553943508, 4.85309666406198]
};


describe(`Map snepshot test`, () => {
  it(`Should map and point render correctly`, () => {
    const tree = renderer
            .create(<Map
              place = {
                PLACE
              }
            />)
              .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
