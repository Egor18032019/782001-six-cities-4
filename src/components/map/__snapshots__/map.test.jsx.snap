// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map snepshot test Should map and point render correctly 1`] = `
<section
  className="cities__map map"
>
  <div
    id="map"
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  />
</section>
`;
